lib/services/ios-project-service.ts

```diff
@@ -221,11 +231,34 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 				this.$logger.info("The iOS Deployment Target is now 8.0 in order to support Cocoa Touch Frameworks.");
 			}
 
-			let frameworkPath = this.getFrameworkRelativePath(libraryPath);
-			project.addFramework(frameworkPath, frameworkAddOptions);
+			let frameworkRelativePath = this.getLibSubpathRelativeToProjectPath(path.basename(frameworkPath));
+			project.addFramework(frameworkRelativePath, { customFramework: true, embed: true });
 			this.savePbxProj(project).wait();
 		}).future<void>()();
 	}
+	
+	private addStaticLibrary(staticLibPath: string): IFuture<void> {
+		return (() => {
+			this.validateStaticLibrary(staticLibPath).wait(); 
+			// Copy files to lib folder.
+			let libDestinationPath = path.join(this.$projectData.projectDir, path.join("lib", this.platformData.normalizedPlatformName));
+			let headersSubpath = path.join("include", path.basename(staticLibPath, ".a"));
+			this.$fs.ensureDirectoryExists(path.join(libDestinationPath, headersSubpath)).wait();
+			shell.cp("-Rf", staticLibPath, libDestinationPath);
+			shell.cp("-Rf", path.join(path.dirname(staticLibPath), headersSubpath), path.join(libDestinationPath, "include"));
+			
+			// Add static library to project file and setup header search paths
+			let project = this.createPbxProj();
+			let relativeStaticLibPath = this.getLibSubpathRelativeToProjectPath(path.basename(staticLibPath));
```

Comment:
`path.basename(staticLibPath)` looks like used alot, consider introducing a named variable for it.